server/src/infra/repositories/asset-job-status.repository.ts

```diff
@@ -0,0 +1,14 @@
+import { IAssetJobStatusRepository } from '@app/domain';
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { AssetJobStatusEntity } from '../entities';
+
+@Injectable()
+export class AssetJobStatusRepository implements IAssetJobStatusRepository {
+  constructor(@InjectRepository(AssetJobStatusEntity) private repository: Repository<AssetJobStatusEntity>) {}
+
+  async upsert(info: Partial<AssetJobStatusEntity>): Promise<void> {
+    await this.repository.upsert(info, { conflictPaths: ['assetId'] });
+  }
```

Comment:
Can you move this method into the asset repository (similar to exif) and call it upset job status?

Reply:
Ah, yeah. That would result in far less boilerplate. Thanks for the suggestion.